### DISCUSSION TITLE
RedwoodJS & Netlify deploy is failing during migration

### DISCUSSION THREAD

**Discussion by noire-munich** [prisma/prisma#2927] [Q&A]
Hello,

I'm opening an issue here as part of my investigation for one opened on RedwoodJS community: https://community.redwoodjs.com/t/production-database-is-not-building/838

I am currently trying to deploy my first RedwoodJS application on Netlify using Heroku's JAWSMysql plugin for my db.
Locally everything is fine, migrations are created properly, the schema is built and the migration table is populated, up and running as expected.

On production my tables are all created to the exception of _migration table and the process is left hanging for 20 minutes without any output thrown. 

**I am expecting this**
```
Or read about them in the ./migrations/MIGRATION_ID/README.md

🚀    Done with 1 migration in 5.00s.
```

**But only getting this**
```
8:28:42 AM: Or read about them in the ./migrations/MIGRATION_ID/README.md
8:56:13 AM: Build exceeded maximum allowed runtime
```

For the record, as part of my investigation I set `PRISMA_HIDE_UPDATE_MESSAGE` to `1`.

**Comment by noire-munich** [comment]
I triggered a new deploy without removing the schema built: it created the _migration table and inserted a row about the migration with an error saying tables were already there.
However it did not move on to the next step, the process got stuck at the same point.
Can't really call that a progress.

**Comment by janpio** [comment]
Do I understand correctly that you clicked yourself a MySQL database on Heroku via JawsMySQL and copied over that connection string to an env var on Netlify (or put it into your schema file directly)? (This should totally work of course, just making sure I got the stack fully understood.)

**Reply by noire-munich** [reply]
Yes you have: I put the connection string in an ENV var, this part works great I assume as I notice movement on my schema ( tables created, but issues on what happens after the steps of a migration are over )

**Reply by janpio** [reply]
Thanks.

Have you tried connected to said database from localhost already? Meaning, does it only hang if it is being run from Netlify vs. localhost?

**Reply by noire-munich** [reply]
Now I have!
Report ID: 8461

Specifically the CLI shows this:

```
Failure during a migration command: Connector error. (error: Error querying the database: Error querying the database: S
erver error: `ERROR 42S02 (1146): Table 'c7xstd0q7435k9ek._migration' doesn't exist'
   0: migration_core::api::ApplyMigration
           with migration_id="20200701075337"
             at migration-engine/core/src/api.rs:77)
```

How can it actually happen?
